### DISCUSSION TITLE
Build failing on simulator after upgrading all firebase package to 11.1.2

### DISCUSSION THREAD

**Discussion by ANIBIT14** [invertase/react-native-firebase#5066] [Q&A] [answered]
Getting the following error while running on x86 simulator after upgrading firebase module packages and gradle version to 4.0.1 :- Crashlytics could not find the manifest. Not found at /Users/aniruddha/Documents/Work/oneCard app/android/app/build/intermediates/merged_manifests/debug/arm64-v8a/AndroidManifest.xml it is running fine on arm64 emulator and devices but not on x86 emulators

**Comment by mikehardy** [comment, marked as answer]
Hi there!
1) this is an issue you are having not a discussion really, with an issue we would have necessary details to troubleshoot

however

2) It runs in CI all the time, you can check our github actions and see it https://github.com/invertase/react-native-firebase/actions/workflows/tests_e2e.yml (note the *tests* might fail because they are flaky sometimes but the build always works)

so I'm thinking 

3) you have a project-specific problem that no one else is experiencing (certainly, my work projects are all building fine) so you might try `npx react-native-clean-project` and/or https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh 

*then* if you are still having a problem, post a minimal reproducible example of the problem on a new issue and we can triage more

**Reply by ANIBIT14** [reply]
Thanks, found the issue was just going to close this discussion.
